tests/test_features.py

```diff
@@ -129,7 +129,7 @@ def test_cast_to_python_objects_tuple(self):
         casted_obj = cast_to_python_objects(obj)
         self.assertDictEqual(casted_obj, expected_obj)
 
-    def test_cast_to_python_objects_numpy(self):
+    def test_cast_to_python_or_numpy(self):
```

Comment:
We'll need to update this test to make sure that numpy arrays are unchanged instead of being casted to python lists.